Bulls-and-cows/src/main/java/study/OperationType.java

```diff
@@ -0,0 +1,30 @@
+package study;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum OperationType {
+    PLUS("+", Math::addExact),
+    MINUS("-", Math::subtractExact),
+    DIVIDE("*", Math::multiplyExact),
+    MULTIPLY("/", Math::floorDiv);
+
+    private final String operation;
+    private final BiFunction<Integer, Integer, Integer> function;
+
+    OperationType(String operation, BiFunction<Integer, Integer, Integer> function) {
+        this.operation = operation;
+        this.function = function;
+    }
+
+    public Integer apply(Integer bucket, Integer value) {
```

Comment:
```suggestion
    public Integer calculate(Integer bucket, Integer value) {
```
apply는 BiFunction에서 제공하는 네이밍인데 조금 더 명확하게 하면 좋지 않을까요 ?.?